src/chains.ts

```diff
@@ -243,6 +255,10 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
     {
       l2Chain: 'soneium',
       l1StandardBridgeAddress: '0xeb9bf100225c214efc3e7c651ebbadcf85177607',
+    },
+    {
+      l2Chain: 'celo',
+      l1StandardBridgeAddress: '0x9C4955b92F34148dbcfDCD82e9c9eCe5CF2badfe',
```

Comment:
last thing, I cant find in celo docs where this address is listed. Can you share this info?

Reply:
It's here: https://specs.celo.org/core_contracts.html#mainnet

ehterscan: https://etherscan.io/address/0x9C4955b92F34148dbcfDCD82e9c9eCe5CF2badfe

Reply:
sweet, thanks!